src/types/form.ts

```diff
@@ -14,11 +14,7 @@ export interface BaseField {
 }
 
 export interface TextField extends BaseField {
-  type: 'SHORT_ANSWER' | 'LONG_ANSWER'
-}
-
-export interface DateField extends BaseField {
-  type: 'DATE'
```

Comment:
DATE and TIME fields act exactly like SHORT_ANSWER and LONG_ANSWER from what I could see, so it makes more sense to group them together

Reply:
Are their format the same? Would it make sense to add some validation regarding that? I will have to take a deeper look into how google forms handles these type of inputs